lib/vsc/install/testing.py

```diff
@@ -36,7 +36,16 @@
 import re
 import sys
 
-from cStringIO import StringIO
+try:
+    basestring
+except NameError:
+    from past.builtins import basestring
```

Comment:
add comments to make it clear one is for Python 2.x, other is for Python 3.x? same below